### ISSUE TITLE
iptables-restore v1.6.1: iptables-restore: unable to initialize table 'nat'

### ISSUE THREAD

**Issue by leonyan1314** [istio/istio#28897]
I am facing a bug when i inject lstio sidecar to default ns, all of pod can't init for below bug:

[root@ibm-master istio-exc]# kubectl logs webapp-859f59c86d-q6sfl   istio-init
iptables-restore --noflush /tmp/iptables-rules-1605323206851298933.txt962677351
**iptables-restore v1.6.1: iptables-restore: unable to initialize table 'nat'**

Error occurred at line: 1
Try `iptables-restore -h' or 'iptables-restore --help' for more information.
iptables-save 
panic: exit status 2

My Env:
OS: RedHat AS 8
[root@ibm-master istio-exc]# uname -a
Linux master 4.18.0-240.1.1.el8_3.x86_64 #1 SMP Fri Oct 16 13:36:46 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux
[root@ibm-master istio-exc]# kubelet --version
Kubernetes v1.19.4

Any

**Comment by howardjohn** [comment]
Can you give more info about how you installed? Are there any other logs from the init container indicating why it failed?

**Comment by leonyan1314** [comment]
I am so sorry, it was my fault for did not close swap... thanks.

**Comment by zhangzerui20** [comment]
@leonyan1314 I also encountered the same problem. Please tell me how did you solve it?

**Comment by themowski** [comment]
We encountered this issue when deploying Kubeflow v1.3.1 and v1.4.1 through its manifests deployment method, which injects Istio sidecars. Our target environment is a downstream Kubernetes v1.21.9 cluster that is deployed by Rancher v2.6.3-patch1. Each node in our cluster has AlmaLinux 8.5 (one of the CentOS 8 derivatives) as its OS. SELinux and firewalld are both disabled on our nodes.

We found a solution in the comments on related issue #23009. The answer was to ensure that a set of specific kernel modules is loaded. For more details, see my reply on the other issue: https://github.com/istio/istio/issues/23009#issuecomment-1049166026
